databases/sql-1.md

```diff
@@ -55,21 +55,21 @@ Most software applications store data of some sort -- user data, content, etc. D
 
 1. Work through the [Codecademy SQL Tutorial](https://www.codecademy.com/learn/learn-sql) **Section 1 (Manipulation)**:
 
-1. PostgreSQL is a popular, open-source version of SQL. Try it out:
+2. PostgreSQL is a popular, open-source version of SQL. Try it out:
    - In Terminal, type `psql` to use PostgreSQL.
-   - Try adding a table with the command `CREATE TABLE food (name TEXT, calories INTEGER);`
-   - See that the table was created by typing `\dt` to see the list of all existing tables.
-   - Try adding some data: `INSERT INTO food VALUES ('pizza', 500);`
+   - Try adding a table with the command `CREATE TABLE food (id INTEGER, name TEXT, calories INTEGER);`
+   - See that the table was created by typing `.\dt` to see the list of all existing tables.
+   - Try adding some data: `INSERT INTO food VALUES ("pizza", 500);`
```

Comment:
double-quotes aren't the same as single-quotes in SQL so this needs to be changed back to single quotes.

Also, now that there is an "id" column in the food table, this insert statement should include a value for id as well.

You should be able to double-check that your changes are correct by running the new commands in SQL and seeing that they work -- let me know if you hit any problems doing that!